protected/humhub/modules/space/behaviors/SpaceModelModules.php

```diff
@@ -34,19 +36,21 @@ class SpaceModelModules extends Behavior
     public function getAvailableModules()
     {
 
-        if ($this->_availableModules !== null) {
-            return $this->_availableModules;
+        if ($this->availableModules !== null) {
+            return $this->availableModules;
```

Comment:
Similar blocks of code found in 2 locations. Consider refactoring 
from https://codeclimate.com/github/humhub/humhub/pull/2951

Reply:
Should be fixed by #2953 as indicated.

Reply:
Where? `protected/humhub/modules/space/behaviors/SpaceModelModules.php` != ` protected/humhub/modules/user/behaviors/UserModelModules.php`